src/sidebar/templates/sidebar-content.html

```diff
@@ -46,5 +46,5 @@
   thread="vm.rootThread">
 </thread-list>
 
-<loggedout-message ng-if="vm.shouldShowLoggedOutMessage()" on-login="vm.onLogin()">
-</loggedout-message>
+<logged-out-message ng-if="vm.shouldShowLoggedOutMessage()" on-login="vm.onLogin()">
```

Comment:
A number of the changes here are me changing from `loggedout` to `logged-out`, as I personally find that easier to read.